### PR TITLE
sentry configurations

### PR DESCRIPTION
**Pull request by maggarwal13**
There are some changes in log4j2.xml file as well.

### CODE REVIEW

File: whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private static final Logger LOGGER = LoggerFactory.getLogger(SentryConfigurations.class);
+
+    private final ApplicationVersion applicationVersion;
+    private final String sentryDsn;
+    private final String environment;
+
+    @Autowired
+    public SentryConfigurations(
+            @Value("${sentry.dsn:}") final String sentryDsn,
+            @Value("${sentry.environment:}") final String environment,
```
Comment (eshryane) [inline whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java:25]: `sentry.environment` is optional, should you check for empty string and return if not defined? Or should we assume a default value (such as production)?

Comment (maggarwal13) [reply whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java:25]: Ok I will add a check I guess making default as production is error prone as if someone forgets to add environment property in predev errors will be reported as production then

Comment (maggarwal13) [reply whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java:25]: Done

Comment (eshryane) [reply whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java:25]: Thanks!

**Review by eshryane — APPROVED** [review]

**Pull request merged**